pkg/machines/libvirt-common.es6

```diff
@@ -179,7 +179,7 @@ export function parseDumpxml(dispatch, connectionName, domXml, id_overwrite) {
 
     const name = domainElem.getElementsByTagName("name")[0].childNodes[0].nodeValue;
     const id = id_overwrite || domainElem.getElementsByTagName("uuid")[0].childNodes[0].nodeValue;
-    const osType = osTypeElem.nodeValue;
+    const osType = osTypeElem.getAttribute("arch");
```

Comment:
So if I have
```xml
  <os>
    <type arch='x86_64' machine='pc-i440fx-2.10'>hvm</type>
    <boot dev='hd'/>
  </os>
```
Then this changes `osType` from "hvm" to "x86_64". If you really want to get the architecture here, please also rename `osType` to `arch` to not confuse reading other parts of the code that use this value.

Reply:
Done.